Properties/AssemblyInfo.cs

```diff
@@ -6,11 +6,11 @@
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 [assembly: AssemblyTitle("GitTfs.GitExtensions.Plugin")]
-[assembly: AssemblyDescription("")]
+[assembly: AssemblyDescription("A GitExtensions plugin for git-tfs.")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyCompany("Nathan Roe")]

```

Comment:
Who is "Nathan Roe" ? Could you change that for something more like "Git-Tfs team"?